src/main/java/io/github/thebusybiscuit/slimefun4/implementation/items/tools/ExplosiveTool.java

```diff
@@ -192,7 +194,19 @@ private BlockExplodeEvent createNewBlockExplodeEvent(
         List<Block> blocks,
         float yield
     ) {
-        // TODO: Support older vers
-        return new BlockExplodeEvent(block, block.getState(), blocks, yield, ExplosionResult.DESTROY);
+        String[] version = Bukkit.getBukkitVersion().split("-");
+
+        if (Integer.parseInt(version[1]) >= 21) {
```

Comment:
Use Slimefun.getMinecraftVersion(). I think Walshy hasn't added 1.21 to that, so u gotta do it first.